backend/spec/requests/api/v1/profiles_spec.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Api::V1::Profiles' do
```

Comment:
There are no constant like this, but you have `Api::V1::ProfilesController`

As this test for the `ProfilesController` we need to use correct constant here